test/integration/manageiq_service/meta/main.yml

```diff
@@ -0,0 +1 @@
+---
```

Comment:
These files should be in the `targets` directory if they can be run via our CI,

`test/integration/targets/manageiq_service/`